docs/tips/14-communicate-between-components.md

```diff
@@ -10,30 +10,40 @@ next: expose-component-functions.html
 For parent-child communication, simply [pass props](/react/docs/multiple-components.html).
 
 For child-parent communication:
-Say your `GroceryList` component has a list of items generated through an array. When a list item is clicked, you want to display its name:
+Say your `GroceryList` component has a list of `Item`s generated through an array. When a list item is clicked, you want to display its name:
 
 ```js
-var handleClick = function(i, props) {
-  console.log('You clicked: ' + props.items[i]);
-}
+var GroceryList = React.createClass({
+  show: function(e){
+    console.log(e);
+  },
+  render: function(){
+    return(
+      <div>
+        {this.props.items.map(function(item, index){
+          return <Item key={index} value={item} log={this.show}/>
+        }.bind(this))}
```

Comment:
Please don't use `.bind(this)` here. `map()` takes an extra argument for scope, or you can pass an arrow function to `map()`